README.md

```diff
@@ -308,9 +308,10 @@ line-length = 88
 target-version = ['py37']
 include = '\.pyi?$'
 exclude = '''
-
-(
-  /(
+# A regex preceded with ^/ will apply only to files in the root of
```

Comment:
```suggestion
# A regex preceded with ^/ will apply only to files and directories in the root of
```

Given that the regex this comment is preceding is only excluding directories, it would be a good idea to add that usage in the comment.

Reply:
Good point, done!